br-webui/_includes/header.liquid

```diff
@@ -55,7 +55,7 @@
                 <ul class="toggle">
                     <label class="switch">
                         <input id='advanced-checkbox' type="checkbox">
-                        <span class="slidernew round"></span>
+                        <span class="slidernew round" onClicked=onToggleAdvancedSwitch></span>
```

Comment:
This does not work because the DOM event is called `onclick` not `onClicked`:
https://www.w3schools.com/jsref/event_onclick.asp

Compare to my suggestion/example here:
https://github.com/bluerobotics/companion/pull/139/files#diff-02839b9128bd02932a355480c5b1d0cdR58

or to the way we have done this in nearby code:
```
$ grep -R onclick
views/mavproxy.liquid:                        onclick="restartMAVProxy()">Restart MAVProxy</button>
views/mavproxy.liquid:                        disabled onclick="resetOptions()">
views/routing.liquid:			remove.onclick = function() { removeEndpoint(endpoint.id) };
views/routing.liquid:				button.onclick = function () {
views/routing.liquid:			<button type="button" class="btn btn-primary" onclick="saveAll()">Save Config</button>
views/routing.liquid:			<button type="button" class="btn btn-primary" onclick="loadAllSoft()">Load Config (Soft)</button>
views/routing.liquid:			<button type="button" class="btn btn-primary" onclick="loadAll()">Load Config</button>
```

Remember also to add parenthesis to call the function.